tests/extensions/pdf/test_exporter.py

```diff
@@ -0,0 +1,93 @@
+import os
+import shutil
+
+import pytest
+
+from mfr.extensions.pdf import settings
+from mfr.extensions.pdf import exceptions
+from mfr.extensions.pdf import PdfExporter
+
+BASE = os.path.dirname(os.path.abspath(__file__))
+
+
+@pytest.fixture
+def directory(tmpdir):
+    return str(tmpdir)
+
+
+@pytest.fixture(scope="function", autouse=True)
+def setup_filesystem(directory):
+    shutil.rmtree(directory)
+    os.makedirs(directory, exist_ok=True)
+
+
+class TestPdfExporter:
+    '''
+    Opening and veryifying pdfs with report lab is a paid feature
```

Comment:
Nitpick: "verifying".  Also, "report lab" could probably use a little more context here, it's only mentioned/used in requirements.txt and the renderer file.

Reply:
reworded/fixed typos/added new typos
complete